addons/arsenal/arsenale/fnc_arsenalUSA.sqf

```diff
@@ -278,7 +278,6 @@ private _allgemein_uniformen = [
     "rhs_uniform_cu_ucp",
     "rhs_uniform_FROG01_d",
     "rhs_uniform_FROG01_wd",
-    "rhs_uniform_g3_mc",
```

Comment:
gibt es dafür einen Grund? das das nicht Multicam?

Reply:
Ja genau. Multicam und OCP ähneln sich zu 99%. Bei Multicam sorgt der zu große Arm-Patch jedoch für grafische Glitches, wenn ein Plattenträger mit Armschutz drüber getragen wird. Aktuell spielt das tatsächlich noch keine Rolle, außer um Platz/Übersicht im Arsenal zu gewinnen, aber ich gehe davon aus, dass die erweiterte Weste, die in diesem PR erscheint, bald auch in OCP sich gewünscht wird. 
Davon abgesehen, weiß ich sowieso nicht, warum wir noch Multicam+OCP in der Auswahl haben, da wir bisher immer ähnliche Skins auf nur einen reduziert haben.